lib/connect/api.js

```diff
@@ -81,27 +81,8 @@ let request = exports.request = function (context, method, path, data) {
 let withUserConnections = exports.withUserConnections = co.wrap(function * (context, appName, flags, allowNone, heroku) {
   // global_config();
 
-  if (flags && flags.resource) {
-    cli.hush('Calling Platform API for resource id')
-
-    return co(function * () {
-      let addon
-      try {
-        addon = yield heroku.apps(appName).addons(flags.resource).info()
-      } catch (err) {
-        if (err.statusCode === 404) {
-          err.body.message = "Connection with resource name '" + flags.resource + "' not found"
-        }
-        throw err
-      }
-      flags.resource_id = addon.id
-      delete flags['resource']
-      return yield Promise.resolve(withUserConnections(context, appName, flags, allowNone, heroku))
-    })
-  }
-
   cli.hush('Retrieving user connections')
-  let response = yield request(context, 'GET', '/api/v3/connections', {deep: true, app: appName, resource_id: flags.resource_id})
```

Comment:
Can you skip the tests that are failing? Further, I'm wondering if we need to add checking that `resource` is never not provided.

Reply:
I've added a fix for the tests that should be related to this. Also, it's allowed to omit `--resource`. An app-level `connect:info` command returns info about all the connections attached to that app. We could change that behavior, but that's how it works today, and it continues to work that way after this change.